example/lib/common/ui/organisms/video_tile.dart

```diff
@@ -58,9 +58,14 @@ class _VideoTileState extends State<VideoTile> {
         _meetingStore.localPeer?.role.permissions.removeOthers ?? false;
     bool changeRolePermission =
         _meetingStore.localPeer?.role.permissions.changeRole ?? false;
-    bool isSimulcastEnabled =
-        (_meetingStore.localPeer?.role.publishSettings?.simulcast?.video !=
-            null);
+    bool isSimulcastEnabled = (context
+            .read<PeerTrackNode>()
+            .peer
+            .role
+            .publishSettings
+            ?.simulcast
+            ?.video !=
+        null);
```

Comment:
Correct check for whether simulcast is enabled is this >  `role.publishSettings?.simulcast?.video.layers != null`

The `layers` list will be non-null when Simulcast is enabled for role